draft-bft-rats-kat.md

```diff
@@ -145,49 +161,63 @@ related entities.
 ~~~
 {: #fig-arch title="Architecture"}
 
-The Presenter generates the IK, which contains a public key (pkT) and a private key (skT), for example
-using the following API call:
+The Presenter triggers the generation of the IK. The IK
+consists of a public key (pkT) and a private key (skT).
```

Comment:
just a curiosity, what does `T` means in `[ps]kT` ?

Reply:
The text was initially in the TLS attestation draft and there it was called TLS Identity Key. The T was related to the TLS part. Now, we are only talking about the Identity Key anymore and hence the abbreviation is less useful....